src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2110,7 +2110,7 @@ class RideConstructionWindow final : public Window
             const auto& ted = GetTrackElementDescriptor(trackType);
             int32_t trackTypeCategory = ted.Definition.type;
 
-            if (trackTypeCategory == TRACK_NONE)
+            if (ted.Description == STR_EMPTY)
```

Comment:
Only special track elements have strings, AFAIK, and only those special elements will show up in the special elements dropdown.